modules/exploits/linux/redis/redis_replication_cmd_exec.rb

```diff
@@ -13,9 +13,9 @@ class MetasploitModule < Msf::Exploit::Remote
 
```

Comment:
Since this is already being referenced and used, might be worthwhile to add a `moved_from` in here:
```suggestion
include Msf::Module::Deprecated

moved_from "exploit/linux/redis/redis_unauth_exec"
```

Reply:
That's reasonable. Thanks!